src/devices/video/ppu2c0x.cpp

```diff
@@ -445,6 +471,24 @@ void ppu2c0x_rgb_device::init_palette_tables()
 	}
 }
 
+void ppu2c04_clone_device::init_palette_tables()
+{
+	/* clone HW doesn't use color emphasis bits.
+	   however, it does have two separate palettes: colors 0-63 for background, and 64-127 for sprites
+	   (although the tile and sprite colors are identical in the Vs. SMB bootleg ROMs)
+	*/
+	for (int color_num = 0; color_num < 64*2; color_num++)
+	{
+		/* A7 line on palette ROMs is always high, color bits are in reverse order */
+		u8 color = m_palette_data[color_num | 0x80];
+		int R = bitswap<3>(color,      0, 1, 2);
+		int G = bitswap<3>(color >> 3, 0, 1, 2);
+		int B = bitswap<2>(color >> 6, 0, 1);
```

Comment:
This would be clearer like this
```
		int R = bitswap<3>(color, 0, 1, 2);
		int G = bitswap<3>(color, 3, 4, 5);
		int B = bitswap<2>(color, 6, 7);
```